encryptionkeyrotation/README.md

```diff
@@ -12,15 +12,15 @@
 ## Objective
 
 Define an extension to the CSI specification that will enable storage vendors
-(SP) to develop controllers/plugins that can rotate the Encryption Keys (KEK)
+(SP) to develop controllers/plugins that can rotate the Key Encryption Keys (KEK)
 of existing volumes that are encrypted.
 
 ### Goals in MVP
 
 The new extension will define a procedure that
 
 - can be called for existing encrypted volumes
-- functions for Block mode volumes
+- functions for RBD Block and filesystem access mode volumes
```

Comment:
Please remove references to RBD, as the specification is independent of the storage system that is used.

Reply:
Ack.